src/clients.rs

```diff
@@ -238,7 +238,7 @@ impl StateClient for State {
 
         self.status("Getting cached block height...");
         let psk = vk.public_spend_key();
-        let last_height = state.cache.last_height(psk)?;
+        let mut last_height = state.cache.last_height(psk)?;
```

Comment:
This seems changed just for debug purpose, right? In case remove it

Reply:
No, we update it in the while loop, see line 259, and we also print it.

This is not for debug purpose, it's for the user – see line `254` that uses a `self.status` and then we display later on the last block fetched.

In the future I would like to show some progress in that regards, but at least now the user knows from block to which block is synced.